lib/cfn-nag/jmes_path_discovery.rb

```diff
@@ -3,13 +3,13 @@ def initialize(rule_registry)
     @rule_registry = rule_registry
   end
 
-  def warning(id:, jmespath:, message:)
+  def warning(id:, message:)
     @rule_registry.definition(id: id,
                               type: Violation::WARNING,
                               message: message)
   end
 
-  def failure(id:, jmespath:, message:)
```

Comment:
i should just pull all the jmespath crud out.... it was an after thought and i don't really test it or think about it so it's pretty much dead code that distracts